index.js

```diff
@@ -71,9 +164,16 @@ module.exports = function(app) {
         default: false,
         title: "Reject self signed and invalid server certificates"
       },
+      selectedOption: {
+        type: "string",
+        title: "Data to send to remote server",
+        enum: ["1) vessel.self", "2) all deltas", "3) self paths in JSON format", "4) all deltas + JSON"],
```

Comment:
"self vessel" or "vessels.self"

I think these are a little hard to understand. This is not just "type of data", it is also "what data". Should there be two selections, one for "what" and another for "format"?

Reply:
Typo. Will be corrected.